packages/tools/src/utilities/math/polyline/getSpacingAndXYDirections.ts

```diff
@@ -0,0 +1,86 @@
+import { StackViewport, VolumeViewport } from '@cornerstonejs/core';
+import { vec3 } from 'gl-matrix';
+import type { Types } from '@cornerstonejs/core';
+
+const EPSILON = 1e-3;
+
+/**
+ * Gets the desired spacing for points in the polyline for the
+ * `PlanarFreehandROITool` in the x and y canvas directions, as well as
+ * returning these canvas directions in world space.
+ */
+const getSpacingAndXYDirections = (
+  viewport: Types.IStackViewport | Types.IVolumeViewport,
+  subPixelResolution: number
```

Comment:
Needs documentation for the function parameters

Reply:
Done.